static/locale/zh_Hant/LC_MESSAGES/django.po

```diff
@@ -4266,7 +4266,7 @@ msgstr ""
 
 #: zerver/views/streams.py:50
 #, python-brace-format
-msgid "User not authorized to execute queries on behalf of '{principal}'"
+msgid "Unable to find user '{principal}'"
```

Comment:
Language translation happens automatically, so we don't need to change the text. So you should remove these changes from all language file, it will be done automatically. To read more about how language translation works, refer [this](https://zulip.readthedocs.io/en/latest/translating/translating.html#translation-process) docs.